sketch.js

```diff
@@ -1,223 +1,69 @@
-//Kanye: Me vs We 
+//Kanye: Me vs We
 
-function setup() {
-  createCanvas(1120,600);
-  background(51);
-}
+var meWords = ["I","me","Kanye","West"]
+var weWords = ["you","we","youre","they", "he","she","them"]
 
-function draw() {
-  fill('hotpink');
-  textSize(25);
-  textFont("Helvetica")
-  text("Kanye: Me vs We", 40, 50);
-  textSize(12);
-  //text("an infographic", 50, 80);
-  
-  kanye();
-  albumNames();
-  splitWords();
+// Splits a string into an array of words
+var splitWords = function(aStringOfLyrics) {
```

Comment:
2 things
- consistency when defining functions
- hoisting. The `var` key word pulls the definitions to the top so you don't have to worry about calling the function before it's defined.